GitUI/UserControls/RevisionGrid.cs

```diff
@@ -2761,18 +2762,33 @@ private void CheckUncommitedChanged(string filtredCurrentCheckout)
                 stageCount = "";
             }
 
-            //Add working directory as virtual commit
+            var userName = Module.GetEffectiveSetting(SettingKeyString.UserName);
+            var userEmail = Module.GetEffectiveSetting(SettingKeyString.UserEmail);
+
+            // Add working directory as virtual commit
             var workingDir = new GitRevision(Module, GitRevision.UnstagedGuid)
             {
+                Author = userName,
```

Comment:
Fake info should be avoided. More in issue comments.

Reply:
I disagree that it is a fake info - it is the name of the individual making changes. This information comes from the gitconfig. Same applies to the email.